custom_components/tuya_local/devices/blitzwolf_bwsh5_humidifier.yaml

```diff
@@ -1,4 +1,3 @@
-# sample DPS {'1': True, '4': 'health', '5': True, '10': 24, '12': 75, '13': 55, '14': 77, '18': 'c', '19': 'cancel', '20': 0, '21': True, '22': 0, '23': 'level_1', '26': True, '35': True} 
 name: BlitzWolf BWSH5 humidifier
```

Comment:
Move this name under products, and add a more generic name here (just Humidifier for example)

Reply:
How will people know the difference if there is few products that are recognized as humidifier if all humidifiers has same name?

Reply:
If someone has multiple humidifiers, they will by default receive a name with a number suffix.  It is up to the user to then name the devices in a way that they will recognise them.